src/dlgTriggerEditor.cpp

```diff
@@ -4048,8 +4101,8 @@ void dlgTriggerEditor::saveKey()
     }
     QString command = mpKeysMainArea->lineEdit_command->text();
     QString script = mpSourceEditor->toPlainText();
-    if( pItem )
-    {
+
+
         int triggerID = pItem->data(0, Qt::UserRole).toInt();
         TKey * pT = mpHost->getKeyUnit()->getKey( triggerID );
```

Comment:
Perhaps we should change `pT` to `pKey` (or key for consistency elsewhere?) - I'm guessing `pT` was pointer to Trigger as that was the first item to be in the editor wasn't it - though I suppose it could also be using `T` in the same way that that meta-symbol is used to represent the Type of a class in a template?